lib/Common/Core/FaultInjection.cpp

```diff
@@ -1398,11 +1405,22 @@ namespace Js
             return EXCEPTION_CONTINUE_EXECUTION;
         }
 
-        struct AutoValue {
-            AutoValue() { inExceptionHandler = true; }
-            ~AutoValue() { inExceptionHandler = false; }
-        } autoVal;
-        FaultInjection::Global.FaultInjectionAnalyzeException(ExceptionInfo);
+        inExceptionHandler = true;
+
+        __try 
```

Comment:
can it be handled with standard try, catch ?

i.e. if you use try/catch instead, it wouldn't catch stack overflow ?